site/docs/components/static-list/examples.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+  <LivePreview componentName="static-list" exampleName="DefaultSingle" >
+
+## Default Single
+
+Default `Static list` with a single item.
+
+  </LivePreview>
```

Comment:
The point of the List is to present multiple items, so I would move the Multiple example to the first example.
An example with one item is not that useful, I would remove.

if we have one example, we could drop the prefix Default.
There are not different variants, there is just a Static List with children and the content dictates the form.